Makefile

```diff
@@ -263,3 +263,10 @@ check-goreleaser-tool-check:
 # Check that all the tools are installed.
 .PHONY: check-tools
 check-tools: check-docker-tool-check check-docker-buildx-tool-check check-docker-compose-tool-check check-protoc-tool-check check-golangci-lint-tool-check check-mockgen-tool-check check-goreleaser-tool-check
+
+.PHONY: fmt-local
+fmt-local:
```

Comment:
nit: how about fmt-diff which may be more descriptive? (since in my mind local makes me think there's a fmt for remote)

Reply:
I have replaced the old `fmt` with the new `fmt` because of https://github.com/guacsec/guac/pull/1507#discussion_r1418968856. So the name won't be `fmt-local` or `fmt-diff`, and will just be `fmt`.